crates/router/src/connector/powertranz.rs

```diff
@@ -103,20 +111,22 @@ impl ConnectorCommon for Powertranz {
             .response
             .parse_struct("PowertranzErrorResponse")
             .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
+        let first_error = response.errors.first();
+        let code = first_error.map(|error| error.code.clone());
+        let message = first_error.map(|error| error.message.clone());
 
         Ok(ErrorResponse {
             status_code: res.status_code,
-            code: response.code,
-            message: response.message,
-            reason: response.reason,
+            code: code.unwrap_or_else(|| consts::NO_ERROR_CODE.to_string()),
+            message: message.unwrap_or_else(|| consts::NO_ERROR_MESSAGE.to_string()),
+            reason: None,
```

Comment:
Why are we sending reason "None" when the message is errors.first(). Why can't we concatenate the other errors send them in reason?